source/06_faq/index.rst

```diff
@@ -80,4 +80,31 @@ The curve of learning rate is shown in the figure below
 
 .. image:: images/Q5_lr_scheduler.png
    :align: center
-   :height: 250
\ No newline at end of file
+   :height: 250
+
+Q6: How to understand the printed [EVALUATOR] information
+***************************************************************
+
+:A6:
+
+We print out the evaluation information of ``evaluator`` in `interaction_serial_evaluator.py <https://github.com/opendilab/DI-engine/blob/main/ding/worker/collector/interaction_serial_evaluator.py#L253>`_ ,
+including ``env``, ``final reward``, ``current episode``, which represent the ``eval_env`` index (``env_id``) corresponding to the current completed episode (in which ``timestep.done=True``),
+the final reward of the completed episode, and how many episodes it was evaluated by the ``evaluator``, respectively.
+
+In some cases, different evaluator environment (abbreviated as ``eval_env``) in the ``evaluator`` may collect episodes with different lengths. For example, suppose we collect 12 episodes through ``evaluator``, but only have 5 ``eval_env``,
+i.e. setting ``n_evaluator_episode=12, evaluator_env_num=5`` in config,
+If we do not limit the number of evaluation episodes in each ``eval_env``, it is likely that we will get many episodes with short lengths.
+As a result, the average reward obtained in this evaluation phase will be biased and cannot fully reflect the performance of the current policy (Only reflects the performance on episodes with shorter lengths).
+
+To address this issue, we propose to utilize the `VectorEvalMonitor <https://github.com/opendilab/DI-engine/blob/main/ding/worker/collector/base_serial_evaluator.py#L78>`_ class.
+In this class, We specify the number of episodes each ``eval_env`` needs to evaluate in `here <https://github.com/opendilab/DI-engine/blob/main/ding/worker/collector/base_serial_evaluator.py#L103>`_.
+and we use `dict <https://github.com/opendilab/DI-engine/blob/main/ding/worker/collector/base_serial_evaluator.py#L110>`_ to store the rewards of the episodes evaluated on each ``eval_env``. Note that, for each ``eval_env``, we
```

Comment:
don't need to talk too much about implementation details like `dict` and `deque`